update_registry.py

```diff
@@ -0,0 +1,124 @@
+import os
+import yaml
+import pytest
+from network_wrangler import ProjectCard
+import pandas as pd
+
+card_dir = os.path.join(".", "projects")
+filename_registry = "registry.csv"
+filename_config = "registry_config.yml"
+
+card_filenames = []
+for (dirpath, dirnames, filenames) in os.walk(card_dir):
+    for filename in filenames:
+        name, extension = os.path.splitext(filename)
+        if extension in [".yml", ".yaml"]:
+            card_filenames.append(os.path.join(card_dir, filename))
+    break
+
+registry_df = pd.read_csv(filename_registry)
+node_df = registry_df[registry_df["node_or_link"] == "node"]
+link_df = registry_df[registry_df["node_or_link"] == "link"]
+
+with open(filename_config, "r") as config_file:
+    config_dict = yaml.safe_load(config_file)
+
+for card_filename in card_filenames:
+    write_updated_card = False
+    card = ProjectCard.read(card_filename, validate=False)
+    card_dict = card.__dict__
+    card_index = 0
+
+for card_filename in card_filenames:
```

Comment:
This needs to be broken up into 3-5 different functions which get called (and are documented).